package.json

```diff
@@ -130,6 +130,7 @@
     "rimraf": "6.0.1",
     "ts-node": "10.9.2",
     "tsup": "8.2.4",
+    "tsx": "4.19.0",
```

Comment:
a bit of a bummer that it will install both even if you only ever use one of them, but better DX than needing to install it yourself to use the tool, or to do what I had been doing and registering `@swc-node/register` myself.

Reply:
This are only `devDependencies`, and I plan to use `tsx` for some other things anyway 😇